includes/class-accredible-learndash-admin-menu.php

```diff
@@ -23,7 +23,7 @@ public static function add() {
 			add_menu_page(
 				'Accredible LearnDash Add-on',
 				'Accredible LearnDash Add-on',
-				'administrator',
+				'manage_options',
```

Comment:
To fix the following PHPCS violation.

<img width="892" alt="Screenshot 2024-10-03 at 11 03 29" src="https://github.com/user-attachments/assets/6dd77c7f-be91-4947-93e4-a2e8fede9eef">

We need to provide not a role but a capability for the 3rd argument of `add_menu_page` and `add_submenu_page`.

c.f.
- https://developer.wordpress.org/reference/functions/add_menu_page/
- https://wordpress.org/documentation/article/roles-and-capabilities/